### ISSUE TITLE
Reproducing GPipe accuracy results

### ISSUE THREAD

**Issue by sg2** [kakaobrain/torchgpipe#1]
Hi, thank you for creating this code. Have you been able to reproduce any of the GPipe accuracy claims with torchgpipe? If so, could you make that training code available?

**Comment by sublee** [comment]
Hi @sg2, do you mean the previous SOTA model, that is AmoebaNet-B (6, 512) with GPipe? If so, we haven't reproduced it yet.

**Comment by sublee** [comment]
Hi @sg2, thanks for your question.

We haven't reproduced the accuracy of any model due to focusing on performance and memory usage instead. But we have been evaluating the transparency of GPipe several times. 

However, we are planning to compare accuracy with and without GPipe on a model, such as ResNet-101. I'll let you know when we prepared the result.

**Comment by sublee** [comment]
My coworker @wbaek has done the experiments for ResNet-101 Accuracy Benchmark. You can get the reproducible code in [examples/resnet101_accuracy_benchmark](https://github.com/kakaobrain/torchgpipe/tree/master/examples/resnet101_accuracy_benchmark).

**Comment by sg2** [comment]
Thanks @sublee and @wbaek!
